src/main/java/toothless/ui/Ui.java

```diff
@@ -0,0 +1,96 @@
+package toothless.ui;
+
+import toothless.task.Task;
+
+import java.util.Scanner;
+
+/**
+ * Represents the user interface of the application.
+ */
+public class Ui {
+    private final static String TOOTHLESS_LOGO =
+            """
+                     _____            _   _     _
+                    |_   _|___   ___ | |_| |__ | | ___  ___ ___
+                      | |/ _ \\ / _ \\| __| '_ \\| |/ _ \\/ __/ __|
+                      | | (_) | (_) | |_| | | | |  __/\\__ \\__ \\
+                      |_|\\___/ \\___/ \\__|_| |_|_|\\___||___/___/
+                    """;
+    private final static String DIVIDER = "*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*\n";
+    private final Scanner sc = new Scanner(System.in);
+
+    /**
+     * Reads the command entered by the user.
+     * @return The command entered by the user.
+     */
+    public String readCommand() {
+        return sc.nextLine();
+    }
+
+    /**
+     * Prints the welcome message.
+     */
+    public void welcome() {
+        System.out.println("Welcome to the dragon's den!\n" + TOOTHLESS_LOGO);
+        System.out.println("Toothless:\n" +
+                "Greetings, Dragon Rider!\n\n" +
+                "I'm Toothless, your friendly dragon companion.\n" +
+                "What adventure shall we embark on today?\n\n" + DIVIDER);
```

Comment:
Perhaps the line break should be done before the operator to improve readability?